### PR TITLE
Feature/error handler service

### PR DESCRIPTION
**Pull request by KyleSaunders1310**

### CODE REVIEW

File: data-accessors/projects.data-accessor.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    constructor(
+        private mongodbService: MongodbService
+    ) {}
+
+    public async getProjects(
+        query: Filter<WithId<Document>> = {},
+        sort: Sort = {},
+        limit = 0
+    ): Promise<MongoDocument[]> {
+        return await this.mongodbService.get(DatabaseCollections.Projects, query, sort, limit)
```
Comment (Solenoden) [inline data-accessors/projects.data-accessor.ts:17]: Theres no need to return the result of promise. Just return the promise

File: services/mongodb.service.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+export class MongodbService {
+    private database: Db
+
+    private async connectToDatabase(): Promise<void> {
+        if (this.database)
+            return
+
+        // eslint-disable-next-line max-len
+        const uri = 'mongodb+srv://admin:admin@portfoliowebsite-db.j0puk.mongodb.net/myFirstDatabase?retryWrites=true&w=majority'
```
Comment (Solenoden) [inline services/mongodb.service.ts:15]: You should have this sensitive info in the environment variables

**Review by Solenoden — CHANGES_REQUESTED** [review]

File: services/mongodb.service.ts (hunk 2 above)
Comment (KyleSaunders1310) [reply services/mongodb.service.ts:15]: I haven't implemented Environment variables yet.

File: data-accessors/projects.data-accessor.ts (hunk 1 above)
Comment (KyleSaunders1310) [reply data-accessors/projects.data-accessor.ts]: Fixed it.

**Review by Solenoden — APPROVED** [review]

**Pull request merged**